Lombiq.Hosting.Tenants.Management/Controllers/ShellSettingsEditorController.cs

```diff
@@ -119,8 +102,8 @@ public async Task<IActionResult> Edit(ShellSettingsEditorViewModel model)
         // not save settings that has a key with multiple sections, see
         // https://github.com/OrchardCMS/OrchardCore/issues/14481. Once this is fixed, we can get rid of the locking and
```

Comment:
Don't forget to remove the workaround, since this issue is now fixed.

Reply:
https://github.com/OrchardCMS/OrchardCore/issues/15184

I didn't find a way to make this actually work. Also it broke after 1.8. The problem seems to be different from the described in the issue.

My problem was that when we wanted to delete keys because they are no longer in the editor I couldn't save them.
No matter what kind of `Update()` or `Save` method I called. I could only delete after referencing `ShellConfiguration[key]` directly instead of using a `Dictionary`. Tested in the Source aswell.

Reply:
Document https://github.com/OrchardCMS/OrchardCore/issues/15184 here instead, then, expanding on the current docs, so during a later upgrade we can understand what needs to be changed.

Reply:
I added a comment in our code with link to the issue and followed up there.

Reply:
I removed the workaround after PPing with Krisztián he confirmed that the way it is now should have been the original way.